web/testing/prediction-ui/index.html

```diff
@@ -47,17 +47,17 @@
           ? window.location.href.substr(0, window.location.href.lastIndexOf('/')+1)
           : window.location.href;
 
-        var modelStub = {'id': 'example.en.trie',
+        var modelStub = {'id': 'example.en.wordlist',
           languages: ['en'],
-          path: (pageRef + "simple-en-trie.js")
```

Comment:
Will this conflict with #1837?

Reply:
It will now ... I will update this accordingly.